server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/CloudFoundryFileCreator.java

```diff
@@ -0,0 +1,72 @@
+package org.opentosca.toscana.plugins.cloudfoundry;
+
+import java.io.IOException;
+import java.util.Map;
+
+import org.opentosca.toscana.core.plugin.PluginFileAccess;
+
+public class CloudFoundryFileCreator {
+
+    private static PluginFileAccess fileAccess;
+    private CloudFoundryApplication app;
+
+    public CloudFoundryFileCreator(PluginFileAccess fileAccess, CloudFoundryApplication app){
+        this.fileAccess=fileAccess;
+        this.app=app;
+    }
+
+    public void createFiles() throws IOException{
+        createManifest();
+        createBpConfig();
+        createDeploySh();
+    }
+
+    private void createManifest() throws IOException{
+        String manifestContent;
+        manifestContent=createHeadMani();
+        manifestContent+=createEnv();
+        manifestContent+=createService();
+        fileAccess.access("/manifest.yml").append(manifestContent).close();
+    }
+
+    private String createHeadMani(){
+        return "applications: \n- name: "+app.getAppName();
+    }
+
+    private String createEnv(){
+        String envBlock;
+        envBlock="\n  env:";
```

Comment:
In general, i'd put the `\n` at the end of `creatHeadManifest` -- maybe this is only my personal preference.